pkg/start/start.go

```diff
@@ -211,10 +211,11 @@ func (o *Options) run(ctx context.Context, controllerCtx *Context, lock *resourc
 	// moved back inside RunOrDie
 	// TODO: properly wire ctx here
 	go leaderelection.RunOrDie(context.TODO(), leaderelection.LeaderElectionConfig{
-		Lock:          lock,
-		LeaseDuration: leaseDuration,
-		RenewDeadline: renewDeadline,
-		RetryPeriod:   retryPeriod,
+		Lock: lock,
+		// TODO set the slower lease values after we're able to reliably shut down gracefully.  This fixes the slow status update for now.
```

Comment:
Set constants instead of using defaults.

I would prefer halving / thirding our current vs this more dramatic change.